static/js/misc.js

```diff
@@ -51,7 +51,18 @@
 
                     label.appendChild(input);
                     label.appendChild(document.createTextNode(' '));
-                    label.appendChild(document.createTextNode(this.textContent));
+
+                    // Hack to bold the definition since we have to strip rST formatting
+                    const [headText, tailText] = this.textContent.split(':', 2);
+                    if (tailText) {
+                        const bold = document.createElement('b'),
+                              defined = document.createTextNode(`${headText}:`);
+                        bold.appendChild(defined);
+                        label.appendChild(bold);
+                    }
+
+                    label.appendChild(document.createTextNode(tailText || headText));
+                    label.normalize();
```

Comment:
`normalize` isn't in-place, I think this line is useless. Out of curiosity, why did you need to normalize it to begin with?